data/sql/updates/pending_db_world/rev_1617016392531315494.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1617016392531315494');
+
+UPDATE creature_template SET InhabitType = 3 WHERE (entry = 3282); -- Venture Co. Mercenary
+UPDATE creature_template SET InhabitType = 3 WHERE (entry = 3283); -- Venture Co. Enforcer
+UPDATE creature_template SET InhabitType = 3 WHERE (entry = 3284); -- Venture Co. Drudger
+UPDATE creature_template SET InhabitType = 3 WHERE (entry = 3445); -- Supervisor Lugwizzle
+UPDATE creature_template SET InhabitType = 3 WHERE (entry = 5836); -- Engineer Whirleygig
```

Comment:
you can combine them with 
```sql
... WHERE `entry` IN (x, y, z, ...);
```

Reply:
Yes, I just thought I'd do it this way so its easy for anyone to see which creatures I actually update, saw it like that in other PRs. 

Should I update it?

Reply:
yes, you can still have the comments:

```sql
UPDATE creature_template SET InhabitType = 3 WHERE entry IN (
  3282, -- Venture Co. Mercenary
  3283, -- Venture Co. Enforcer
  3284, -- Venture Co. Drudger
  3445, -- Supervisor Lugwizzle
  5836  -- Engineer Whirleygig
); 
```